src/data/real/basic.lean

```diff
@@ -88,6 +88,7 @@ instance : monoid ℝ             := by apply_instance
 instance : comm_semigroup ℝ     := by apply_instance
 instance : semigroup ℝ          := by apply_instance
 instance : has_sub ℝ            := by apply_instance
+instance : module ℝ ℝ           := by apply_instance
```

Comment:
Contrary to the comment on line 74, your added instance is not just for short-circuiting.
```suggestion
-- Promote a computable `module` instance deriving from the multiplicative structure.
-- Otherwise, we'd get a noncomputable instance going via `normed_space.to_module`.
instance : module ℝ ℝ           := by apply_instance
```

Reply:
This comment actually applies to all of these instances

Reply:
I've added a similar comment above.